workers/tasks/export/html.tsx

```diff
@@ -91,10 +91,10 @@ const addAttrsToNodes = (newAttrs, matchNodeTypes, nodes) =>
 		return node;
 	});
 
-const getCitationLinkage = (unstructuredValue, structuredValue) => {
+const getCitationLinkage = (unstructuredValue, structuredValue, nodeId = null) => {
 	const digest = digestCitation(unstructuredValue, structuredValue);
 	return {
-		inlineElementId: `citation-${digest}-inline`,
+		inlineElementId: `citation-${digest}${nodeId && '-' + nodeId}-inline`,
```

Comment:
```suggestion
		inlineElementId: `citation-${digest}${nodeId ? '-' + nodeId : ''}-inline`,
```

I can't tell if this is just a cosmetic fix for links or if it's an issue but this version will avoid printing `'null'`

Reply:
No, this is needed -- there are some cases where we may not want to pass in the ID.